Assignment 3.Rmd

```diff
@@ -26,12 +28,15 @@ The algorithm will treat each row as a value belonging to a person, so we need t
 
 ```{r}
 
-K2 <- 
+K2 <- select(K1, motivation1,motivation2,motivation3,motivation4,motivation5 )
 
 ```
 
 It is important to think about the meaning of missing values when clustering. We could treat them as having meaning or we could remove those people who have them. Neither option is ideal. What problems do you foresee if we recode or remove these values? Write your answers below:
 
+If we remove these values, it will affect the results of our calculation e.g. the average
+
```

Comment:
Can you elaborate a little on what you mean here. What does effecting the average mean for your conclusions?